prosody/rootfs/defaults/conf.d/visitors.cfg.lua

```diff
@@ -187,18 +187,23 @@ Component '{{ $VISITORS_MUC_PREFIX }}.v{{ $VISITOR_INDEX }}.{{ $VISITORS_XMPP_DO
 
     {{ if $ENABLE_RATE_LIMITS -}}
     -- Max allowed join/login rate in events per second.
-	rate_limit_login_rate = {{ $RATE_LIMIT_LOGIN_RATE }};
-	-- The rate to which sessions from IPs exceeding the join rate will be limited, in bytes per second.
-	rate_limit_session_rate = {{ $RATE_LIMIT_SESSION_RATE }};
-	-- The time in seconds, after which the limit for an IP address is lifted.
-	rate_limit_timeout = {{ $RATE_LIMIT_TIMEOUT }};
-	-- List of regular expressions for IP addresses that are not limited by this module.
-	rate_limit_whitelist = {
-      "127.0.0.1";
-      {{ range $index, $cidr := (splitList "," $RATE_LIMIT_ALLOW_RANGES | compact) -}}
-      "{{ $cidr }}";
-      {{ end -}}
-    };
+    rate_limit_login_rate = {{ $RATE_LIMIT_LOGIN_RATE }};
```

Comment:
Can you please avoid the unnecessary formatting changes here?

Reply:
I do agree this is mostly formatting, but before this PR there was mixed indentation (tabs/spaces) in this file.
For next time I assume the better approach would be to create a seperate PR for this?

Reply:
Generally speaking, yes. But since you are fixing spacing, let's leave it!